components/Card/fragments/MicroService/interfaces.ts

```diff
@@ -1,5 +1,6 @@
 interface MicroServiceProps {
-    serviceName: string
+    serviceName: string,
+    isMicroservice: boolean,

```

Comment:
`isMicroservice` kann optional sein
Wenn es optional wäre, müssten wir die variable nicht immer geben, und als hätte es false als value


```suggestion
    serviceName: string,
    isMicroservice?: boolean,
```